package.json

```diff
@@ -57,7 +57,7 @@
     "cypress:prd:watch": "npm run cypress -- open",
     "predevelop": "npm run create:config",
     "develop": "npm-run-all build:curriculum -p develop:*",
-    "develop:client": "cd ./client && npm run develop",
+    "develop:client": "npm run build:curriculum && cd ./client && npm run develop",
     "develop:server": "npm run predevelop && cd ./api-server && npm run develop",
```

Comment:
Since build:curriculum isn't horrendously slow, how about we just run it and create:config before both? It's somewhat wasteful, but probably results in fewer weird things happening.

Reply:
If you want?

It might just be Tom and I 🤷‍♂️ - we always develop the server first, then the client. So, having both do both is hardly an issue.

---

Shall I add `build:curriculum` to `predevelop`, and add `predevelop` to `develop:client`?

Reply:
On reflection, it might not be sensible to have two commands both trying to build the curriculum in parallel. Your approach seems fine, though.